### PR TITLE
OAuth configuration steps updated

### PR DESCRIPTION
**Pull request by prateeksainilambdatest**
### What does this PR do?

OAuth configuration steps updated

### CODE REVIEW

File: lambdatest/README.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ Here's how you can track incidents in Datadog with LambdaTest:
 2. Select settings from the menu bar.
 3. Click on integrations under settings.
```
Comment (anawishnoff) [inline lambdatest/README.md:30]: When a user clicks "Connect Accounts" on the datadog tile, they'll be taken to the onboarding_url that you specified when creating the OAuth client, which I believe is this Lambdatest integrations page. If so, we can remove lines 28-30 and replace with the following:

```suggestion
1. Click "Connect Accounts" to begin authorization of the LambdaTest integration. You will be taken to the Integrations page in LambdaTest.
```

File: lambdatest/README.md [hunk 2]
```diff
@@ -29,7 +29,7 @@ Here's how you can track incidents in Datadog with LambdaTest:
...
 4. To integrate Datadog, click on install next to Push to Datadog.
```
Comment (anawishnoff) [inline lambdatest/README.md:31]:
```suggestion
2. In LambdaTest, click "Install" next to **Push to Datadog**. 
```

File: lambdatest/README.md [hunk 3]
```diff
@@ -29,7 +29,7 @@ Here's how you can track incidents in Datadog with LambdaTest:
...
-5. Enter your [Datadog API and application key][2].
+5. Click on Authorize button to complete the integration process.
```
Comment (anawishnoff) [inline lambdatest/README.md:32]:
```suggestion
5. Click **Authorize** to complete the integration process.
```

File: lambdatest/README.md [hunk 4]
```diff
@@ -29,7 +29,7 @@ Here's how you can track incidents in Datadog with LambdaTest:
...
 6. A confirmation email is sent. Confirm the integration in the email.
```
Comment (anawishnoff) [inline lambdatest/README.md:33]:
```suggestion
6. A confirmation email is sent once the integration configuration is complete.
```

File: lambdatest/README.md (hunk 2 above)
Comment (prateeksainilambdatest) [reply lambdatest/README.md]: Updated accordingly

**Review by bgoldberg122 — APPROVED** [review]

File: lambdatest/README.md [hunk 5]
```diff
@@ -25,13 +25,11 @@ All configuration happens on the LambdaTest Dashboard. See the [LambdaTest-Datad
...
 
-1. Log in to your LambdaTest account on the LambdaTest website.
-2. Select settings from the menu bar.
-3. Click on integrations under settings.
-4. To integrate Datadog, click on install next to Push to Datadog.
-5. Enter your [Datadog API and application key][2].
-6. A confirmation email is sent. Confirm the integration in the email.
-7. Once Datadog is integrated with your LambdaTest account, start logging bugs and performing cross-browser testing.
+1. Click "Connect Accounts" to begin authorization of the LambdaTest integration. You will be taken to the Login page in LambdaTest.
+2. Log in to your LambdaTest account on the LambdaTest website and you'll be redirected to your Datadog application.
```
Comment (anawishnoff) [inline lambdatest/README.md:29]:
```suggestion
2. Log in to your LambdaTest account on the LambdaTest website and you'll be redirected to the Datadog authorization page.
```

**Review by anawishnoff — APPROVED** [review]

**Review by buraizu — CHANGES_REQUESTED** [review]
Thanks, just made some suggestions for consistency and to better align with our [style guide](https://github.com/DataDog/documentation/blob/master/CONTRIBUTING.md).

File: lambdatest/README.md [hunk 6]
```diff
@@ -25,13 +25,11 @@ All configuration happens on the LambdaTest Dashboard. See the [LambdaTest-Datad
...
+3. Click **Authorize** to complete the integration process.
+4. A confirmation email is sent once the integration configuration is complete.
+5. Once Datadog is integrated with your LambdaTest account, start logging bugs and performing cross-browser testing.
```
Comment (anawishnoff) [inline lambdatest/README.md:33]:
```suggestion

### Uninstallation

To ensure this integration is fully uninstalled, revoke authorization in the Configure tab of this tile or in [OAuth Apps](https://app.datadoghq.com/organization-settings/oauth-applications) within Organization Settings in Datadog. 

Additionally, ensure that all API keys associated with this integration have been disabled by searching for the integration name on the [API Keys management page](https://app.datadoghq.com/organization-settings/api-keys).
```

Please also add these OAuth-specific instructions under the Configuration section. Thanks!

**Review by anawishnoff — APPROVED** [review]
This is good to go, thanks @prateeksainilambdatest !

**Review by FlorentClarret — APPROVED** [review]

File: lambdatest/README.md [hunk 7]
```diff
@@ -25,13 +25,17 @@ All configuration happens on the LambdaTest Dashboard. See the [LambdaTest-Datad
...
 
-1. Log in to your LambdaTest account on the LambdaTest website.
-2. Select settings from the menu bar.
-3. Click on integrations under settings.
-4. To integrate Datadog, click on install next to Push to Datadog.
-5. Enter your [Datadog API and application key][2].
-6. A confirmation email is sent. Confirm the integration in the email.
-7. Once Datadog is integrated with your LambdaTest account, start logging bugs and performing cross-browser testing.
+1. Click **Connect Accounts** to begin authorization of the LambdaTest integration from the Login page in LambdaTest.
+2. Log in to your LambdaTest account on the LambdaTest website and you'll be redirected to the Datadog authorization page.
```
Comment (buraizu) [inline lambdatest/README.md:29]:
```suggestion
2. Log in to your LambdaTest account on the LambdaTest website to be redirected to the Datadog authorization page.
```

Comment (buraizu) [reply lambdatest/README.md:29]: Please remove the future tense here as mentioned in our style guide:
https://github.com/DataDog/documentation/blob/master/CONTRIBUTING.md#tense

**Review by buraizu — CHANGES_REQUESTED** [review]

**Pull request merged**